app/scripts/modules/core/pipeline/config/validation/pipelineConfig.validator.spec.ts

```diff
@@ -88,7 +89,7 @@ describe('pipelineConfigValidator', () => {
   }));
 
   beforeEach(mock.inject((_pipelineConfigValidator_: PipelineConfigValidator,
-                          _pipelineConfig_: PipelineConfigService,
+                          _pipelineConfig_: any,
```

Comment:
So this is kind of badass/PITA.  The compiler is actually checking that the `spyOn` calls refer to methods that actually exist on the object which the dev claims they're being spied upon.  In this case, the `spyOn` calls are actually spying on the `.provider('pipelineConfig')` which is not TS and `angular.auto.IProvideService` is not templatized so it doesn't know the type.  Either way, however, I think PipelineConfigService wasn't correct, originally.